sdk/storage/AzureStorageBlob/README.md

```diff
@@ -61,13 +61,22 @@ az storage account create -n my-storage-account-name -g my-resource-group
 ### Create the client
 The Azure Storage Blobs client library for iOS allows you to interact with blob storage containers and blobs.
 Interaction with these resources starts with an instance of a [client](#client). To create a client object, you will
-need the storage account's blob service account URL and a credential that allows you to access the storage account:
+need a credential that allows you to access the storage account:
+
+```swift
+import AzureStorageBlob
+
+let credential = ...
+let client = StorageBlobClient(credential: credential)
+```
+
+Certain types of credentials may also require you to provide the storage account's blob service account URL:
 
 ```swift
 import AzureStorageBlob
 
 let accountUrl = "https://<my-storage-account-name>.blob.core.windows.net/"
-let credential = MSALCredential(...)
+let credential = ...
 let client = StorageBlobClient(accountUrl: accountUrl, credential: credential)
```

Comment:
The only public initializer that follows this signature is the MSALCredential, and restoration ID is required.

Reply:
I wasn't sure if we needed to include the rest of the parameters here, since this section is just focusing on the types of credentials. What if I append an ellipses to these examples, and then add a section about the restoration ID (which we need to document anyway)?

```swift
let client = StorageBlobClient(accountUrl: accountUrl, credential: credential, ...)
```

Reply:
Yep, I think that works. It at least communicates that there is "other stuff" needed.

Reply:
Otherwise it becomes a document bug when someone posts how they copied the code that looked complete and it didn't compile. :)

Reply:
OH that's an excellent point, I hadn't considered that folks would just copy and paste lines out of these docs. Will absolutely remember that for future docs writing!